examples/POS-client/ERC1155/depositSingle.js

```diff
@@ -7,8 +7,14 @@ const execute = async () => {
     const tx = await maticPOSClient.depositSingleERC1155ForUser(
       config.root.DERC1155,
       config.user.address,
-      config.user.tokenId,
-      config.user.amount
+      '123',
```

Comment:
can we comment besdiee hardcoded value, as what they represents . e.g - what is '123'

Reply:
done